lhotse/audio.py

```diff
@@ -93,6 +94,13 @@ def load_audio(
         else:  # self.type == 'file'
             samples, sampling_rate = read_audio(source, offset=offset, duration=duration)
 
+        # A useful case of resampling here is for the opus-format audios. Opus-format audio would be decoded at 48kHz
+        # by force, with the original sampling rate being ignored.
+        if target_sampling_rate is not None and sampling_rate != target_sampling_rate:
```

Comment:
We shouldn’t do it this way. The changes to audio.py are duplicating existing Lhotse’s functionality (`recordings.resample(target_sampling_rate)`), and in a way that is incompatible with the rest of the code base (how would e.g. Cut know that it has to specify a target_sampling_rate when load_audio() is called on it?)